arm-containerregistry/2017-06-01-preview/swagger/containerregistry.json

```diff
@@ -1068,17 +1084,9 @@
           "readOnly": true
         },
         "provisioningState": {
```

Comment:
No rationale. These services are not doing the right thing and whenever they send a PR for any update, it will be/should be called out to fix.